content/bn/docs/contribute/_index.md

```diff
@@ -0,0 +1,187 @@
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H,M,Q,N,O,P,V grey
+class S,T,U spacewhite
+class first,second,third white
+{{</ mermaid >}}
+***চিত্র - একজন নতুন অবদানকারীর জন্য শুরু করা***
+
+উপরের চিত্রটি নতুন অবদানকারীদের জন্য একটি রোডম্যাপের রূপরেখা দেয়৷ আপনি `Sign up` এবং `Review` এর জন্য কিছু বা সমস্ত ধাপ অনুসরণ করতে পারেন। এখন আপনি 'Open PR'-এর অধীনে তালিকাভুক্ত কিছু সহ আপনার অবদানের উদ্দেশ্যগুলি অর্জনকারী PR খুলতে প্রস্তুত। আবার,আমরা প্রশ্ন সবসময় স্বাগত জানাই!
+
+
+
+কিছু কাজের জন্য Kubernetes সংস্থায় আরও বিশ্বাস এবং আরও অ্যাক্সেসের প্রয়োজন।
+এ সম্পর্কে আরো বিস্তারিত জানার জন্য [SIG ডক্সে অংশগ্রহণ করে](/docs/contribute/participate/) দেখুন
+ভূমিকা এবং অনুমতি সমুহ।
+
+
+
+
+## আপনার প্রথম অবদান 
+
+আপনি আগে থেকে বেশ কয়েকটি ধাপ পর্যালোচনা করে আপনার প্রথম অবদানের জন্য প্রস্তুত করতে পারেন। নীচের চিত্রটি ধাপগুলিকে রূপরেখা দেয় এবং বিশদগুলি অনুসরণ করে৷
+
+<!-- See https://github.com/kubernetes/website/issues/28808 for live-editor URL to this figure -->
+<!-- You can also cut/paste the mermaid code into the live editor at https://mermaid-js.github.io/mermaid-live-editor to play around with it -->
+
+{{< mermaid >}}
+flowchart LR
+    subgraph second[প্রথম অবদান]
+    direction TB
+    S[ ] -.-
+    G[অন্যদের থেকে পুল রিকোয়েস্ট পর্যালোচনা করুন<br>K8s সদস্যদের মধ্য থেকে] -->
+    A[K8s/website চেক করুন<br> সমস্যা তালিকার জন্য<br>ভাল প্রথম পুল রিকোয়েস্ট (good first PRs)] --> B[একটি পুল রিকোয়েস্ট খুলুন!!]
+    end
+    subgraph first[প্রস্তাবিত প্রস্তুতি]
+    direction TB
+       T[ ] -.-
+       D[অবদান ওভারভিউ পড়ুন] -->E[K8s বিষয়বস্তু পড়ুন<br>এবং শৈলী গাইড]
+       E --> F[Hugo পাতা সম্পর্কে জানুন<br>বিষয়বস্তুর প্রকার<br>এবং শর্টকোড]
+    end
+    
+
+    first ----> second
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,D,E,F,G grey
+class S,T spacewhite
+class first,second white
+{{</ mermaid >}}
+***চিত্র - আপনার প্রথম অবদানের জন্য প্রস্তুতি***
+
+
+- [কন্ট্রিবিউশন ওভারভিউ](/docs/contribute/new-content/overview/) পড়ুন এবং
+  আপনি  যেসব উপায়ে অবদান রাখতে পারেন সেসব সম্পর্কে জানুন । 
+- [`kubernetes/website` সমস্যা তালিকা](https://github.com/kubernetes/website/issues/) চেক করুন যা
+  ভালো এন্ট্রি পয়েন্ট তৈরি করে এমন সমস্যার জন্য।
+- [GitHub ব্যবহার করে একটি পুল অনুরোধ খুলুন](/docs/contribute/new-content/open-a-pr/#changes-using-github)
+  বিদ্যমান ডকুমেন্টেশনে এবং GitHub-এ ফাইল করা সমস্যা ( filing issues in GitHub) সম্পর্কে আরও জানুন।
+- অন্য Kubernetes সম্প্রদায়ের সদস্যসদের  থেকে
+  সঠিকতা এবং ভাষার জন্য [পুলের অনুরোধ পর্যালোচনা করুন](/docs/contribute/review/reviewing-prs/)  
+- Kubernetes [content](/docs/contribute/style/content-guide/)  এবং [শৈলী নির্দেশিকা](/docs/contribute/style/style-guide/) পড়ুন
```

Comment:
I'd expect to see “content” localized. For example, does  “ওয়েব সামগ্রী” make sense?  (that's a machine translation&mdash;I don't speak Bangla).